collections/ansible_collections/cloudkit/config_as_code/README.md

```diff
@@ -0,0 +1,225 @@
+# Ansible Collection - cloudkit.config_as_code
+
+This role defines the configuration as code of AAP to run the Ansible playbooks
+that are part of CloudKit.
+
+## Credentials
+
+Since AAP runs in an Openshift cluster, we expect the required credentials to
```

Comment:
As long as we're capitalizing this word, we may as well get the fully correct "OpenShift" here and throughout the PR.

Reply:
done!